fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisTaskInfoBuilder.java

```diff
@@ -30,19 +32,21 @@ public class AnalysisTaskInfoBuilder {
     private String catalogName;
     private String dbName;
     private String tblName;
-    private Set<String> partitionNames;
+    private Map<String, Set<String>> colToPartitions;
```

Comment:
What's the reason for using `Map`  here?  Each task is a one-one mapping to B

Reply:
This is true for task, but it may be a many-to-many relationship for Job. It was changed to Map for ease of processing, keeping the operations on job/task uniform.